common/utils/__tests__/validator-utils.test.js

```diff
@@ -1,63 +1,56 @@
-import { isMinPasswordStrength, isValidZipcode } from '../validator-utils';
+import { isMinPasswordStrength } from '../validator-utils';
+import mockPassword from '../../../test-utils/mockGenerators/mockPassword';
 
 describe('Validator Utilities', () => {
-  describe('isValidZipcode', () => {
-    it('should return true for any string', () => {
-      expect(isValidZipcode('testString')).toStrictEqual(true);
-    });
-
-    it('should return false when passed an empty string', () => {
-      expect(isValidZipcode('')).toStrictEqual(false);
-    });
-
-    it('should return false when passed undefined', () => {
-      expect(isValidZipcode()).toStrictEqual(false);
-    });
```

Comment:
why remove these?

Reply:
`isValidZipCode()` only checks to ensure that the user doesn't enter an empty string or string of just spaces. I added the `trim()` to the Yup call on zipcode to check this just as we did with `firstName` and `lastName`. There is no need for `isValidZipCode()` and therefore no need for the unit tests.